docs/examples/en/loaders/GLTFLoader.html

```diff
@@ -20,6 +20,11 @@ <h1>[name]</h1>
 		textures, skins, skeletons, morph targets, animations, lights, and/or cameras.
 		</p>
 
+		<p>
+			[name] uses [page:ImageBitmapLoader] whenever possible. Please keep in mind that image bitmaps are not automatically GC-collected when they are not referenced anymore.
+			They require a special handling during the disposal process. More information in the [link:https://threejs.org/docs/#manual/en/introduction/How-to-dispose-of-objects How to dispose of objects] guide.
```

Comment:
Be advised that image bitmaps are not automatically GC-collected when they are no longer referenced, and they require special handling during the disposal process.